javaee/impl/src/main/java/org/jboss/forge/addon/javaee/validation/ui/ValidationNewAnnotationCommandImpl.java

```diff
@@ -46,19 +38,13 @@ public Metadata getMetadata(UIContext context)
       return Metadata.from(super.getMetadata(context), getClass())
                .name("Constraint: New Annotation")
                .description("Create a Bean Validation constraint annotation")
-               .category(Categories.create(super.getMetadata(context).getCategory(), "Bean Validation"));
+               .category(super.getMetadata(context).getCategory());
```

Comment:
You can remove this line. Metadata.from already reuses the category from super.getMetadata